packages/file_selector/file_selector_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.3
+
+* Updates minimum supported SDK version to Dart 3.2.
```

Comment:
```suggestion
* Migrates web implementation to `package:web` from `dart:html`.
* Updates minimum supported SDK version to Dart 3.2.
```

Reply:
🤷 – for the most part, this is not a user visible change so I omitted that bit.